### ISSUE TITLE
Wrong Chrome window brought to front

### ISSUE THREAD

**Issue by damithc** [HubTurbo/HubTurbo#450]
In my Win8 computer, I had another Chrome window. HT brings that window to the front instead of the one spawned by HT.

**Comment by rd1992** [comment]
could you exactly describe how you got this behaviour? (i dont have windows 8 so i cant replicate it). Like did you interact with the other window while the HT bview was still firing up? Thank you

**Comment by damithc** [comment]
Yes, that seems to be the case. I tried again and was able to reproduce it.
If I interact with the other chrome window while HT is starting up, this
behavior happens. If I don't touch the other chrome window while HT is
starting up, things seem to be fine.

**Comment by damithc** [comment]
V1.9.2: The windows explorer Window that I used to doubleclick the jar file is getting the focus. To use HT, i have to minimize that explorer Window. If not, it always appears in front. We need a more fail-safe way to identify the bView Window.

**Comment by damithc** [comment]
As a side effect, I can't change projects. When I click on the project switching drop down, focus switch back to Explorer window and come back to HT window, triggering a sync and disabling the project switch :-p 
This is a show stopper. Need to fix before we can release this version.

**Comment by damithc** [comment]
I used the command prompt to launch HT, and now the command window is getting focus. So the behavior is reproducible.

**Comment by rd1992** [comment]
yeah i found the bug. its to do with the window handle. trying a different way on how to get the window handle.

**Comment by damithc** [comment]
Still not fixed (1.9.3). A different window is brought to front. Please test thoroughly before releasing.

**Comment by rd1992** [comment]
checking now. Looks like there are merge conflicts and the code i wrote is not there anymore
